signadot/operator/README.md

```diff
@@ -136,10 +136,11 @@ style resources and are not needed in an installation which uses the new
 |                                          |                                                                                                                                                                                                                                                                              |         |
 ### Istio Parameters
 
-The Signadot Operator manipulates istio objects when istio VirtualServices are applied to workloads in sandboxes.  You can configure the operator to add labels and annotations to these objects when they are in use by the operator.  Note that these labels and annotations are only added if not present when the object comes into use. This can be useful for temporarily disabling CI sync, amongst other possibilities.
+When Istio is enabled (`istio.enabled: true`), the Signadot Operator manipulates Istio VirtualServices by applying new HTTPRoutes for each of the sandboxed workloads. You can configure the operator to add labels and annotations to these objects when they are in use by the operator.  Note that these labels and annotations are only added if not present when the object comes into use. This can be useful for temporarily disabling CI sync, amongst other possibilities.
```

Comment:
I think this problem was there originally at my hand, but I'd suggest trying to rephrase 
`by applying new HTTPRoutes for each of the sandboxed workloads` with  something more precise/informative.  

The balance is hard to find, as being completely precise is maybe too verbose for this context.  But perhaps something along the lines of
```
by applying new HTTPRoutes where appropriate to direct traffic to sandboxed workloads
```
could work.

```

Reply:
I applied the change.